lib/taskana-core/src/test/resources/taskana.properties

```diff
@@ -1,20 +1,64 @@
+#################################################
+# Taskana role configurations
+#################################################
+
+# The list of accessids (could be user or group) to be allowed to access TASKANA as user
 taskana.roles.user=cn=ksc-users,cn=groups,OU=Test,O=TASKANA | teamlead-1 | teamlead-2 | user-1-1 | user-1-2 | user-2-1 | user-2-2 | user-b-1 | user-b-2
+# The list of accessids (could be user or group) to be allowed to access TASKANA as admin
 taskana.roles.admin=admin | uid=admin,cn=users,OU=Test,O=TASKANA
+# The list of accessids (could be user or group) to be allowed to access TASKANA as businessadmin
 taskana.roles.businessadmin=businessadmin | cn=business-admins,cn=groups,OU=Test,O=TASKANA
+# The list of accessids (could be user or group) to be allowed to access TASKANA as monitor
 taskana.roles.monitor=monitor | cn=monitor-users,cn=groups,OU=Test,O=TASKANA
+# The list of accessids (could be user or group) to be allowed to access TASKANA as taskadmin
 taskana.roles.taskadmin=taskadmin
+
+#################################################
+# TASKANA domain configurations
+#################################################
+
+# The comma separated list of administrative domains
 taskana.domains=Domain_A  ,  DOMAIN_B
+# The default domain to create new tasks in if no other domain is specified
+taskana.domains.default=DOMAIN_A
```

Comment:
Should we also add this to the properties of the example?

Reply:
ok.